fe/fe-core/src/main/java/org/apache/doris/common/util/FetchRemoteTabletSchemaUtil.java

```diff
@@ -0,0 +1,323 @@
+            for (Replica replica : tablet.getReplicas()) {
+                Set<Long> tabletIds = beIdToTabletId.computeIfAbsent(
+                                    replica.getBackendId(), k -> Sets.newHashSet());
+                tabletIds.add(tablet.getId());
+            }
+        }
+
+        // build PTabletsLocation
+        List<PTabletsLocation> locations = Lists.newArrayList();
+        List<Backend> coordinatorBackend = Lists.newArrayList();
+        for (Map.Entry<Long, Set<Long>> entry : beIdToTabletId.entrySet()) {
+            Long backendId = entry.getKey();
+            Set<Long> tabletIds = entry.getValue();
+            Backend backend = Env.getCurrentEnv().getCurrentSystemInfo().getBackend(backendId);
+            if (coordinatorBackend.size() < 2) {
+                coordinatorBackend.add(backend);
+            }
+            PTabletsLocation.Builder locationBuilder = PTabletsLocation.newBuilder()
+                                                        .setHost(backend.getHost())
+                                                        .setBrpcPort(backend.getBrpcPort());
+            PTabletsLocation location = locationBuilder.addAllTabletId(tabletIds).build();
+            locations.add(location);
+        }
+        PFetchRemoteSchemaRequest.Builder requestBuilder = PFetchRemoteSchemaRequest.newBuilder()
+                                                                    .addAllTabletLocation(locations)
+                                                                    .setIsCoordinator(true);
+        // send rpc to coordinatorBackend util succeed or 2 times
+        for (Backend be : coordinatorBackend) {
+            try {
+                PFetchRemoteSchemaRequest request = requestBuilder.build();
+                Future<PFetchRemoteSchemaResponse> future = BackendServiceProxy.getInstance()
+                                            .fetchRemoteTabletSchemaAsync(be.getBrpcAddress(), request);
+                PFetchRemoteSchemaResponse response = null;
+                try {
+                    response = future.get(60, TimeUnit.SECONDS);
+                    TStatusCode code = TStatusCode.findByValue(response.getStatus().getStatusCode());
+                    String errMsg;
+                    if (code != TStatusCode.OK) {
+                        if (!response.getStatus().getErrorMsgsList().isEmpty()) {
+                            errMsg = response.getStatus().getErrorMsgsList().get(0);
+                        } else {
+                            errMsg = "fetchRemoteTabletSchemaAsync failed. backend address: "
+                                    + be.getHost() + " : " + be.getBrpcPort();
+                        }
+                        throw new AnalysisException(errMsg);
+                    }
+                    fillColumns(response);
+                    break;
+                } catch (AnalysisException e) {
+                    // continue to get result
+                    LOG.warn(e);
```

Comment:
why continue in `AnalysisException`?

Reply:
When the RPC returns an incorrect result, an AnalysisException will be thrown, we should try sending the RPC again.

Reply:
When will  the RPC returns an incorrect result, and is this a bug?

Reply:
Is not a bug, ColumnPB has a type of bytes, and it needs to be converted to a string. This conversion may throw an AnalysisException.